plugin/pkg/global/extensionlabels/admission.go

```diff
@@ -242,6 +242,8 @@ func addMetaDataLabelsSeed(seed *core.Seed) {
 	if seed.Spec.DNS.Provider != nil {
 		metav1.SetMetaDataLabel(&seed.ObjectMeta, v1beta1constants.LabelExtensionDNSRecordTypePrefix+seed.Spec.DNS.Provider.Type, "true")
 	}
+	metav1.SetMetaDataLabel(&seed.ObjectMeta, v1beta1constants.LabelSeedProvider, seed.Spec.Provider.Type)
+	metav1.SetMetaDataLabel(&seed.ObjectMeta, v1beta1constants.LabelSeedRegion, seed.Spec.Provider.Region)
```

Comment:
I don't see any docs added as part of this PR. How end user or operator is supposed to understand about these labels?